core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -651,7 +652,11 @@ private ExprValue exprConvertTZ(ExprValue startingDateTime, ExprValue fromTz, Ex
    */
   private ExprValue exprDate(ExprValue exprValue) {
     if (exprValue instanceof ExprStringValue) {
-      return new ExprDateValue(exprValue.stringValue());
+      try {
```

Comment:
We should be able to remove the `if` statement and get the same result.  The `try...catch` will cover both cases.

Reply:
I think we still need it because it can be a stringValue or a dateValue.

Reply:
is there a case where an ExprStringValue doesn't have a stringValue()?